src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncLocalsTests.cs

```diff
@@ -1273,5 +1273,263 @@ public class C
                 }, module.GetFieldNames("C.<F>d__0"));
             });
         }
+
+        [WorkItem(9775, "https://github.com/dotnet/roslyn/issues/9775")]
+        [Fact]
+        public void Fixed_Debug()
```

Comment:
`unsafe { ... await expr ... }` results in `error CS4004: Cannot await in an unsafe context`. We have several tests for that error.